test/api/search_controller_test.rb

```diff
@@ -252,11 +252,9 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
     )
     assert_response :unauthorized
 
-    execute_http_request(:get, api_v1_search_bulk_download_path(
-        auth_code: ' ', accessions: study.accession, file_types: file_types)
-    )
-
-    assert_response :unauthorized
+    execute_http_request(:get, api_v1_search_bulk_download_path(accessions: study.accession, file_types: file_types))
+    # response should be successful with no auth_code, since bearer token is valid
+    assert_response :success
```

Comment:
The response should definitely be `401` without an auth code.  If we're responding `200` then we aren't enforcing totats/auth_codes to be present.  Maybe I missed something, but this seems like an obvious error.  If we don't enforce this then users can spam bulk download and rack up enormous egress charges.